.github/workflows/ansys_lab.yml

```diff
@@ -69,8 +69,8 @@ jobs:
       - name: "Push changes"
         shell: bash
         run: |
-          git config --global user.name "rlagha"
+          git config --global user.name "pyansys-ci-bot"
```

Comment:
Did you configure the proper rights to the ci bot?

Reply:
@BClappe it is managed at the organization level exactly for that type of task. I am not the one who gave permissions to the bot but I was told to use it, and considering an update pipeline passed, I'd say it works.

Reply:
As long as the bot is authorized with "write" permissions in your repository, this token should work

Reply:
Anytime!